clients/fides-js/src/components/tcf/FilterButtons.tsx

```diff
@@ -2,35 +2,36 @@ import { h } from "preact";
 import { useState } from "preact/hooks";
 
 interface Filter {
-  name: string;
+  label: string;
+  value: string;
 }
 
-const FilterButtons = ({
+const FilterButtons = <T extends Filter>({
```

Comment:
I refactored this component to use an object instead of just the `name` string. makes comparing LegalBasisEnums a little more type safe